tests/yang_json2xml/yang_json2xml/meta/main.yaml

```diff
@@ -0,0 +1,3 @@
+---
+dependencies:
+  - ../../../network-engine
```

Comment:
This is not needed anymore as the test role is imported in tasks/main.yaml

Reply:
fixed